hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/bucket/OMBucketCreateRequest.java

```diff
@@ -301,6 +309,25 @@ private BucketEncryptionInfoProto getBeinfo(
     return bekb.build();
   }
 
+  /**
+   * Check namespace quota.
+   */
+  private void checkQuotaInNamespace(OmVolumeArgs omVolumeArgs,
+      long allocatedNamespace) throws IOException {
+    if (omVolumeArgs.getQuotaInCounts() > OzoneConsts.QUOTA_RESET) {
+      long usedNamespace = omVolumeArgs.getUsedNamespace();
```

Comment:
Can you use omVolumeArgs.getQuotaInCounts() != OzoneConsts.QUOTA_RESET to replace above condition check? That will be the better check in case maybe OzoneConsts.QUOTA_RESET value can be changed to the positive number.

Reply:
make senses! Done!